schema/examples/proofTypesExample.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://cdn.jsdelivr.net/gh/Hyperproof/hypersync-sdk/schema/proofProviders.schema.json",
+  "$schema": "https://cdn.jsdelivr.net/gh/Hyperproof/hypersync-sdk/schema/proofTypes.schema.json",
```

Comment:
This schema file should have been renamed when we renamed the json files a little while back.